Assets/VRTK/Scripts/Presence/VRTK_BodyPhysics.cs

```diff
@@ -75,6 +75,10 @@ public enum FallingRestrictors
         public int standingHistorySamples = 5;
         [Tooltip("The `y` distance between the headset and the object being leaned over, if object being leaned over is taller than this threshold then the current standing position won't be updated.")]
         public float leanYThreshold = 0.5f;
+        [Tooltip("A GameObject containing a Capsule Collider used as a custom body collider.")]
```

Comment:
Also, is it necessary to pass a GameObject as the custom colliders? Would it not be better to have the type as `Collider`? Something seems mismatched between the variable type and the variable name.

I guess either `GameObject customXXColliderContainer` or `Collider customXXCollider`